third_party/vault/plugin-secrets-kubernetes-reader/values.yaml

```diff
@@ -9,7 +9,7 @@ server:
       readOnly: false
   extraInitContainers:
     - name: kubesecrets-plugin
-      image: ghcr.io/revit13/vault-plugin-secrets-kubernetes-reader:latest
+      image: ghcr.io/the-mesh-for-data/vault-plugin-secrets-kubernetes-reader:latest
```

Comment:
should we always pull the latest? or should it be configured with optional tag? e.g., specific version?

Reply:
There are no versions yet, this is out of scope for this PR.